src/main/kotlin/no/nav/tms/min/side/proxy/Application.kt

```diff
@@ -60,32 +43,30 @@ data class AppConfiguration(
 
     val contentFecther = ContentFetcher(
         proxyHttpClient = proxyHttpClient,
-        eventAggregatorClientId = eventAggregatorClientId,
-        eventAggregatorBaseUrl = eventAggregatorBaseUrl,
-        utkastClientId = utkastClientId,
-        utkastBaseUrl = utkastBaseUrl,
-        personaliaClientId = personaliaClientId,
-        personaliaBaseUrl = personaliaBaseUrl,
-        selectorClientId = selectorClientId,
-        selectorBaseUrl = selectorBaseUrl,
-        varselClientId = varselClientId,
-        varselBaseUrl = varselBaseUrl,
-        statistikkClientId = statistikkClientId,
-        statistikkBaseApiUrl = statistikkBaseUrl,
         oppfolgingClientId = oppfolgingClientId,
         oppfolgingBaseUrl = oppfolgingBaseUrl,
-    )
+        eventAggregator = appResolver.variablesFor("dittnav-event-aggregator"),
+        utkast = appResolver.variablesFor("tms-utkast"),
+        personalia = appResolver.variablesFor(
+            application = "tms-personalia-api",
+            baseUrlPostfix = "/tms-personalia-api"
+        ),
+        selector = appResolver.variablesFor("tms-mikrofrontend-selector"),
+        varsel = appResolver.variablesFor("tms-varsel-api"),
+        statistikk = appResolver.variablesFor("http://tms-statistikk"),
```

Comment:
Blir dobbelt `http://` for statistikk. Denne er også en azure-ting, så den må ha en egen clientId (punktum i stedet for kolon)